src/tests/ftest/util/pool_create_all_base.py

```diff
@@ -119,7 +119,7 @@ def check_pool_full_storage(self, scm_delta_bytes, nvme_delta_bytes=None, ranks=
         if ranks is not None:
             wait_ranks = sorted(ranks)
             data = self.dmg.pool_query(self.pool[pool_idx].identifier, show_enabled=True)
-            got_ranks = sorted(data['response']['enabled_ranks'])
+            got_ranks = sorted(data['response'].get('enabled_ranks'))
```

Comment:
If `enabled_ranks` is not returned this will raise an exception
```
sorted(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'NoneType' object is not iterable
```

We could default it to an empty list so `sorted` works
```suggestion
            got_ranks = sorted(data['response'].get('enabled_ranks', []))
```